misk-events-core/src/main/kotlin/misk/events/Consumer.kt

```diff
@@ -1,8 +1,10 @@
 package misk.events
 
 /** A [Consumer] allows applications to receive events from a source */
+@Deprecated("This API is no longer supported and replaced by go/cash-eventing's client library")
```

Comment:
Should this message better obfuscate Cash specific implementation details before landing in public Misk?